Source/WebCore/Modules/notifications/Notification.cpp

```diff
@@ -128,9 +131,14 @@ Ref<Notification> Notification::create(ScriptExecutionContext& context, Notifica
 
 Ref<Notification> Notification::create(ScriptExecutionContext& context, const URL& registrationURL, const NotificationPayload& payload)
 {
+#if ENABLE(DECLARATIVE_WEB_PUSH)
     Options options;
     if (payload.options)
-        options = { payload.options->dir, payload.options->lang, payload.options->body, payload.options->tag, payload.options->icon, JSC::jsNull(), nullptr, nullptr, payload.options->silent };
+        options = { { }, { }, payload.options->dir, payload.options->lang, payload.options->body, payload.options->tag, payload.options->icon, JSC::jsNull(), nullptr, nullptr, payload.options->silent };
```

Comment:
If you added the new members to the end of the Options struct, would it be possible to reduce the #if/#endif to just the initialisation of options.defaultActionURL?

Reply:
Yah, think that's a good change.

Reply:
...That sadly some of the builds don't accept. *Ah well*